src/components/Modals/Settings/SettingsRouter.tsx

```diff
@@ -17,6 +19,8 @@ export const SettingsRouter = ({ appHistory }: { appHistory: RouteComponentProps
           )}
         />
         <Route path={SettingsRoutes.Languages} component={() => <Languages />} />
+        <Route path={SettingsRoutes.FiatCurrencies} component={() => <FiatCurrencies />} />
+        <Route path={SettingsRoutes.CurrencyFormat} component={() => <CurrencyFormat />} />
```

Comment:
Again a nitpick so no need to commit it but we're not using any prop for these two components, so that just makes it one more level in the JSX tree for no reason, this could either be:

```suggestion
        <Route path={SettingsRoutes.FiatCurrencies} component={FiatCurrencies} />
        <Route path={SettingsRoutes.CurrencyFormat} component={CurrencyFormat} />
```

Or use the conventional children syntax for `<Route />`: https://v5.reactrouter.com/web/guides/primary-components/route-matchers